### DISCUSSION TITLE
DFLib 1.1.0 release

### DISCUSSION THREAD

**Discussion by andrus** [dflib/dflib#419] [Announcements]
Glad to announce the `1.1.0` release of DFLib.

DFLib is an open source pure Java DataFrame library. It is great for writing data transformations, ETL, "notebook" data analysis, visualizing data with charts and many other things. It is developed and supported by [ObjectStyle](https://www.objectstyle.com/) in collaboration with the community.

1.1.0 comes on the heels of 1.0 GA, and is a fully-compatible drop-in replacement. It contains a number of important improvements:

1. A generic `ByteSource` API, that bridges most data "loaders" (CSV, Excel, etc.) with arbitrary sources of binary data (generic URLs, REST APIs, in-memory sources, etc.).
2. Primitive `float` columns and expressions, saving up to 50% of memory in DataFrames with smaller numeric values
3. Support for ECharts `itemStyle` and other styling options (think different colors, dynamic item sizes - essentially an extra visual dimension, etc.)
4. Covering some API gaps (`Exp.shift(..)`, `$bool(..).cumSum()`, `BooleanSeries.firstFalse()`, expressions for `OffsetDateTime` columns)
5. Bug fixes

**Comment by hrstoyanov** [comment]
Love this library!
